src/Client.php

```diff
@@ -2,139 +2,92 @@
 
 namespace League\StatsD;
 
-use League\StatsD\Exception\ConnectionException;
 use League\StatsD\Exception\ConfigurationException;
+use League\StatsD\Exception\ConnectionException;
 
 /**
  * StatsD Client Class
  *
  * @author Marc Qualie <marc@marcqualie.com>
  */
-class Client
+class Client implements StatsDClient
 {
+    /** Instance instances array */
+    protected static array $instances = [];
 
-    /**
-     * Instance instances array
-     * @var array
-     */
-    protected static $instances = array();
+    /** Instance ID */
+    protected string $instanceId;
```

Comment:
It is can be considered as BC break (previous name was a `instance_id`)

Reply:
But adding a property typehints can be considered as BC break too, so this renaming can be approved too.

:thumbsup: